packages/bun-types/package.json

```diff
@@ -1,25 +1,37 @@
 {
   "name": "bun-types",
-  "repository": "https://github.com/oven-sh/bun",
   "license": "MIT",
+  "main": "",
+  "types": "index.d.ts",
+  "description": "Type definitions for Bun, an incredibly fast JavaScript runtime",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oven-sh/bun",
+    "directory": "packages/bun-types"
+  },
+  "files": [
+    "*.d.ts"
+  ],
+  "homepage": "https://bun.sh",
   "dependencies": {
-    "@types/node": "*",
-    "@types/ws": "*",
-    "undici-types": "~5.26.4"
+    "@types/node": "~20.11.3",
+    "@types/ws": "~8.5.10"
   },
   "devDependencies": {
     "@definitelytyped/dtslint": "^0.0.199",
     "@definitelytyped/eslint-plugin": "^0.0.197",
     "prettier": "^2.4.1",
     "typescript": "^5.0.2"
   },
-  "private": true,
```

Comment:
☝️ Exposing this (just confirming).

Reply:
Yes, because this is the package.json that will be published to npm.
Before with bundling the script used to generate a new package.json on the dist folder, now we will just use the one already in the repository.